### DISCUSSION TITLE
记 · 在 AI 公司入职一个月的体验与感悟 | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#298] [General]
# 记 · 在 AI 公司入职一个月的体验与感悟 | 愧怍

愧怍的个人博客

https://kuizuo.cn/blog/experience-of-an-ai-company

**Comment by jkiss** [comment]
年轻真好，从我10年的工作经验来说，Next.js+Tailwind是我目前的最优解，其他的框架一概不想碰。刚毕业进一个氛围好的初创公司挺好的，干几年就可以考虑进个国企了，哈哈哈

**Reply by kuizuo** [reply]
以我性子来看，国企这种相对养老的公司我还是不考虑了。
说到技术栈，我现在只写 TS 和 Node，从 Vue 转 React 了，从开发应用的角度，使用 React 能找到很多解决方案，很多问题或工具选择你是不用过于担心的。
样式我只能说 Tailwindcss yyds！

**Comment by just-to-Cyberspace** [comment]
所以老哥是多大，35了吗？不适合坐班是什么意思，看你们公司加班强度也不大吧，还是我没看仔细。

**Reply by kuizuo** [reply]
那位老哥应该是临近35了，手头接了点外包单，加上有自创业的想法，每天弄到3，4点那种，时间分配不过来。唉都不容易啊。
目前来说公司确实没怎么强度，不加班，周末双休，但我能预感到之后的日子可就不一定了。

**Reply by just-to-Cyberspace** [reply]
初创公司，即便是双休，强度高的话，双休也不一定能完全休息。我之前进的那个C轮公司，也是双休，压力还是有的。

**Reply by just-to-Cyberspace** [reply]
3，4点，太夸张了，那证明本来就没想干很久，就是过渡一下来的。

**Reply by kuizuo** [reply]
> 3，4点，太夸张了，那证明本来就没想干很久，就是过渡一下来的。

确实有这个想法，毕竟年龄和生活压力摆在这。也可能是因为公司急招，所以技术合适就招进来了。

**Comment by albaz64** [comment]
同年毕业正在找工作, 但是现在非常颓废. 虽然弄了一堆项目在本地但是一个都不想碰, 白天睡不醒晚上睡不着.
问问贵司还缺人吗? 感觉我和你还挺相似的, 而且都喝过茶.

**Reply by jkiss** [reply]
来打魔兽，哈哈哈

**Comment by Hare-Success** [comment]
真好，还没学react。

**Reply by jkiss** [reply]
可以直接 next.js 了😎

**Comment by ahaolabel** [comment]
大三正在一家ai初创实习（感觉也不算 北上广深都有部门），实习工资纯赔在上海陆家嘴，不过运气也还好实习老师挺好，leader是业内大牛（top日企首席数据科学家，也很愿意指导人），   在py/cpp/java/计算机基础的坑里狠狠挖 ，手上还有个远程实习正在做   以前也休过学（高中休的身体原因，很久2年以上）只能说家庭可以 父母编制内支撑的了我浪费钱瞎折腾 。

跟博主的路子正好背道而驰，有些感慨。

**Reply by ahaolabel** [reply]
项目到现在反而就一个cpp客户端+自己造轮子的服务器的全栈项目和一群玩具项目，本科可能要好一些相对博主  一所中等的双非一本

**Reply by Hare-Success** [reply]
你评论的为啥会到我的邮箱呢

**Reply by kuizuo** [reply]
> 你评论的为啥会到我的邮箱呢

因为 GIthub Discussions 就是这么设计的😂，只要你有评论过该话题，该话题的后续通知你也将会收到。如果你不需要这个话题的通知，你应该点击右侧的 Unsubscribe

<img width="306" alt="image" src="https://github.com/user-attachments/assets/04974305-0f64-4712-9518-d6fdb4827a82">

**Reply by Hare-Success** [reply]
好吧，我说呢，现在就差搜索没整了，有啥新方案码，申请不申请下来 还没服务器

**Reply by Hare-Success** [reply]
这个取消，还得到你的github里 取消吧。

**Reply by kuizuo** [reply]
> 这个取消，还得到你的github里 取消吧。

你也可以直接在邮件中取消
<img width="387" alt="image" src="https://github.com/user-attachments/assets/f429c799-6373-43af-9bd9-7e8e6c0a5703">

**Comment by Hare-Success** [comment]
哦哦哦  好的了概了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月12日 10:54 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [kuizuo/blog] 记 · 在 AI 公司入职一个月的体验与感悟 | 愧怍 (Discussion #298) |

这个取消，还得到你的github里 取消吧。

你也可以直接在邮件中取消
image.png (view on web)

**Comment by feichuans** [comment]
支持！
